### DISCUSSION TITLE
Make the instance Crypted

### DISCUSSION THREAD

**Discussion by Sebu-Wai-Wou** [AppFlowy-IO/AppFlowy#103] [Ideas]
Wish : Administrator of an appflowy.io instance cannot recover or reset user passwords. This ensures that they cannot access any user data (e.g. the content of documents).

**Comment by lawweiliang** [comment]
I agree with your point, encrypted database like "keepass" and "bitwarden" could avoid other to view our data.

**Reply by appflowy** [reply]
I think we could use SQL cipher to encrypt out the local database.
